packages/web-frontend/src/containers/mobile/EntityHierarchy.js

```diff
@@ -0,0 +1,63 @@
+/**
```

Comment:
Most of this taken from within the old `packages/web-frontend/src/screens/mobile/RegionScreen/index.js`